python/datasources/covid_tracking_project.py

```diff
@@ -14,5 +17,112 @@ def get_id():
     def get_table_name():
         return 'covid_tracking_project'
 
+    @staticmethod
+    def get_standard_columns():
+        """Returns a dict containing conversions from Covid Tracking Project's
+           race categories to their standardized values."""
+        return {
+            'aian': col_std.Race.AIAN.value,
+            'asian': col_std.Race.ASIAN.value,
+            'black': col_std.Race.BLACK.value,
+            'nhpi': col_std.Race.NHPI.value,
+            'white': col_std.Race.WHITE.value,
+            'multiracial': col_std.Race.MULTI.value,
+            'other': col_std.Race.OTHER.value,
+            'unknown': col_std.Race.UNKNOWN.value,
+            'ethnicity_hispanic': col_std.Race.HISP.value,
+            'ethnicity_nonhispanic': col_std.Race.NH.value,
+            'ethnicity_unknown': col_std.Race.ETHNICITY_UNKNOWN.value,
+            'total': col_std.Race.TOTAL.value
+        }
+
     def write_to_bq(self, dataset, gcs_bucket, **attrs):
-        logging.info("Covid Tracking Project write_to_bq not implemented yet")
+        filename = self.get_attr(attrs, 'filename')
+        metadata_table_id = self.get_attr(attrs, 'metadata_table_id')
+        table_name = self.get_attr(attrs, 'table_name')
+
+        df = gcs_to_bq_util.load_csv_as_dataframe(gcs_bucket, filename)
+        self.clean_frame_column_names(df)
+
+        # Massage the data into the standard format.
+        df.drop(columns=['cases_latinx', 'deaths_latinx',
+                         'hosp_latinx', 'tests_latinx'], inplace=True)
+        df = df.melt(id_vars=['date', 'state'])
+        df[['variable_type', 'race_and_ethnicity']] = df.variable.str.split(
+            "_", 1, expand=True)
+        df.drop('variable', axis=1, inplace=True)
+        df.rename(columns={'state': 'state_postal_abbreviation'}, inplace=True)
+        df.replace({'race_and_ethnicity': self.get_standard_columns()},
+                   inplace=True)
+
+        # Get the metadata table
+        metadata = self._download_metadata(metadata_table_id)
```

Comment:
Per a comment in some google doc thread, don't we need a `reports_ethnicity` column too? I'm forgetting the exact details, but I think we discussed needing that information in order to know how to join with ACS?

Reply:
Yes, that information is included in the metadata table.